summarize/src/main.rs

```diff
@@ -18,6 +18,10 @@ struct Opt {
     /// Writes the analysis to a json file next to <file_prefix> instead of stdout
     #[structopt(long = "json")]
     json: bool,
+
+    /// Filters the output rows where "% of total time" > "percent-above"
```

Comment:
I'd provide a longer description here since this will end up in the `--help` flag:

```
Filter the output to items whose self-time is greater than this value.
```